bin/configurable-http-proxy

```diff
@@ -294,6 +298,24 @@ if (!options.authToken) {
   log.warn("REST API is not authenticated.");
 }
 
+// custom headers option
+options.customHeaders = [];
+if (args.customHeaders) {
+  options.customHeaders = args.customHeaders.toString().split(',').map(s => s.trim()).map(header => {
```

Comment:
I don't think simple splitting on commas is going to work, since commas could be in the header values. Maybe try following the [collect example](https://github.com/tj/commander.js/#custom-option-processing) for specifying multiple values for an option, so it would look like:

```
chp --header "a: x,y,z" --header "b: a,b,c"
```

Reply:
Implemented.